cli/cli.go

```diff
@@ -1,29 +1,48 @@
 package main
 
 import (
+	"fmt"
+	"os"
 	"sort"
 	"strconv"
 	"strings"
 
 	toxiproxyServer "github.com/Shopify/toxiproxy"
 	"github.com/Shopify/toxiproxy/client"
 	"github.com/urfave/cli"
-
-	"fmt"
-	"os"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 const (
-	redColor    = "\x1b[31m"
-	greenColor  = "\x1b[32m"
-	yellowColor = "\x1b[33m"
-	blueColor   = "\x1b[34m"
-	cyanColor   = "\x1b[36m"
-	purpleColor = "\x1b[35m"
-	grayColor   = "\x1b[37m"
-	noColor     = "\x1b[0m"
+	RED = iota
+	GREEN
+	YELLOW
+	BLUE
+	CYAN
+	PURPLE
+	GRAY
+	NONE
 )
 
+var colors = map[int]string{
```

Comment:
Are there other advantages to having this hash map besides tty? Why not just have a
```go
func color(color string) string {
    if isTTY {
        return color
    } else {
        return ""
    }
}
```

and then use `color(BLUE)` with string constant `BLUE` rather than enum.